### PR TITLE
iframe upload use post message to get the res content

### PR DESCRIPTION
**Pull request by baiyaaaaa**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [ ] Rebase before creating a PR to keep commit history clear.
* [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: packages/upload/src/iframe-upload.vue [hunk 1]
```diff
@@ -149,23 +107,27 @@ export default {
...
-    },
-
-    getFormNode() {
-      return this.$refs.form;
-    },
-
-    getFormDataNode() {
-      return this.$refs.data;
+    onload(e) {
+      debugger;
```
Comment (Leopoldthecoder) [inline packages/upload/src/iframe-upload.vue]: 😂

**Comment by baiyaaaaa** [comment]
#618

File: packages/upload/src/index.vue [hunk 2]
```diff
@@ -123,7 +120,7 @@ export default {
     },
     handleError(err, file) {
       var _file = this.getFile(file);
```
Comment (furybean) [inline packages/upload/src/index.vue]: 为什么 let 和 var 混用，好像其他人已经不用 var 了 😂

Comment (DanielWLam) [inline packages/upload/src/index.vue]: #618 的问题是在这里修复了是吗？

**Pull request merged**